core/module_specifier.rs

```diff
@@ -31,10 +36,31 @@ impl fmt::Display for ModuleResolutionError {
       ImportPathPrefixMissing => {
         write!(f, "relative import path not prefixed with / or ./ or ../")
       }
+      UnsupportedPathPrefix => write!(
+        f,
+        r"invalid path prefix, must be disk prefix (eg. C:/, C:\\)"
+      ),
```

Comment:
Messages for these two variants should print out specifier and path respectively. Trying to figure it out, but neither `String` nor `PathBuf` implement `Copy` trait... 🤔